lasy/utils/laser_utils.py

```diff
@@ -783,10 +787,10 @@ def export_to_z(dim, grid, omega0, z_axis=None, z0=0.0, t0=0.0, backend="NP"):
             verbose=False,
         )
         # Convert the spectral image to the spatial field representation
-        FieldAxprp.import_field(np.moveaxis(field, -1, 0).copy())
+        FieldAxprp.import_field(xp.moveaxis(field, -1, 0).copy())
```

Comment:
I am currently getting an error of the form:
```
lasy/utils/laser_utils.py:790: in export_to_z
    FieldAxprp.import_field(xp.moveaxis(field, -1, 0).copy())
.../python-3.11/lib/python3.11/site-packages/axiprop/containers.py:369: in import_field
    self.time_to_frequency()
.../python-3.11/lib/python3.11/site-packages/axiprop/containers.py:439: in time_to_frequency
    self.Field_ft[:] = np.fft.ifft(self.Field, axis=0, norm="backward")
```

Reply:
we should get rid of the axiprop `Containers` -- they currently only have CPU implementation for spectral transforms. I assume we are doing FFT now with lasy portable methods, so lets stick to this